### ISSUE TITLE
Support for creating index & migration operations

### ISSUE THREAD

**Issue by osoykan** [Trendyol/stove4k#14]

**Comment by osoykan** [comment]
@AksalBilal @oguzhaneren:

> maybe we can expose couchbase public api to user for advanced operations, with this way we don't bother with abstractions.
```
.couchbase(){ connection->
  connection.bucket("test").createIndex("CREATE INDEX blabla").await()
}
```

**Comment by osoykan** [comment]
It is implemented in #54. Now It can be achieved by using migrations.
